EIPS/eip-3540.md

```diff
@@ -8,7 +8,7 @@ status: Review
 type: Standards Track
 category: Core
 created: 2021-03-16
-requires: 3541, 3860
+requires: 3541, 3860, 4750, 5450
```

Comment:
Isn't this a cyclic dependency?

Reply:
We can resolve this at a later point. Main focus right now is just getting the spec down.